config/optional-features.json

```diff
@@ -1,5 +1,6 @@
 {
   "application-template-wrapper": false,
+  "default-async-observers": true,
```

Comment:
Not sure if this was left out for a reason before? I just kept the blueprint default but I can flip the value if needed.